index.html

```diff
@@ -83,6 +83,54 @@ <h4 class="connect">LET’S CONNECT</h4>
         </div>
       </div>
     </section>
+    <div class="item works-1 pop invisible-pop" id="work-1">
+      <div class="card-content pop-main">
+        <img class="pop-close" src="images/pop-close.svg" alt="">
```

Comment:
- [ ] [REQUIRED] Great job with the mobile popup 👍🏽  However, we have another close button [X] beneath the modal right on top of the `See live` and `See source` buttons that should be removed. Kindly remove this as it is not in the design template provided. 

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="360" alt="Screenshot 2023-04-12 at 11 15 59" src="https://user-images.githubusercontent.com/58692590/231434377-6dbc3fc1-91b0-437e-9bf1-c06fb7a8a413.png"> | <img width="255" alt="Screenshot 2023-04-12 at 11 42 57" src="https://user-images.githubusercontent.com/58692590/231434617-98f78333-4ed9-4d08-a82a-18a25dd3e965.png">